drivers/SmartThings/zwave-siren/src/test/test_philio_sound_siren.lua

```diff
@@ -271,11 +276,6 @@ test.register_coroutine_test(
         })
       )
     )
-    test.wait_for_events()
-    test.mock_time.advance_time(1)
-    test.socket.capability:__expect_send(
-      mock_siren:generate_test_message("main", capabilities.chime.chime.off())
-    )
   end
```

Comment:
this chime off should still be sent if your code is working correctly

Reply:
yes, you're right, thank you